Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM python:2
+
+RUN \
+  apt-get update && \
+  DEBIAN_FRONTEND=noninteractive \
+    apt-get -y install \
+      libprotobuf-dev \
```

Comment:
Pip install of 'mysql-connector' fails when building:
```
  [...]
  Running setup.py install for mysql-connector: started
    Running setup.py install for mysql-connector: finished with status 'error'
    Complete output from command /usr/local/bin/python -u -c "import setuptools, tokenize;__file__='/tmp/pip-build-3o9xdQ/mysql-connector/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmp/pip-xbquKL-record/install-record.txt --single-version-externally-managed --compile:
    running install
    Not Installing MySQL C Extension
    running build
    running build_py
    creating build
    creating build/lib.linux-x86_64-2.7
    creating build/lib.linux-x86_64-2.7/mysql
    copying lib/mysql/__init__.py -> build/lib.linux-x86_64-2.7/mysql
    creating build/lib.linux-x86_64-2.7/mysql/connector
    copying lib/mysql/connector/catch23.py -> build/lib.linux-x86_64-2.7/mysql/connector
    copying lib/mysql/connector/__init__.py -> build/lib.linux-x86_64-2.7/mysql/connector
    copying lib/mysql/connector/connection_cext.py -> build/lib.linux-x86_64-2.7/mysql/connector
    copying lib/mysql/connector/cursor_cext.py -> build/lib.linux-x86_64-2.7/mysql/connector
    copying lib/mysql/connector/abstracts.py -> build/lib.linux-x86_64-2.7/mysql/connector
    copying lib/mysql/connector/utils.py -> build/lib.linux-x86_64-2.7/mysql/connector
    copying lib/mysql/connector/dbapi.py -> build/lib.linux-x86_64-2.7/mysql/connector
    copying lib/mysql/connector/constants.py -> build/lib.linux-x86_64-2.7/mysql/connector
    copying lib/mysql/connector/errorcode.py -> build/lib.linux-x86_64-2.7/mysql/connector
    copying lib/mysql/connector/connection.py -> build/lib.linux-x86_64-2.7/mysql/connector
    copying lib/mysql/connector/conversion.py -> build/lib.linux-x86_64-2.7/mysql/connector
    copying lib/mysql/connector/errors.py -> build/lib.linux-x86_64-2.7/mysql/connector
    copying lib/mysql/connector/network.py -> build/lib.linux-x86_64-2.7/mysql/connector
    copying lib/mysql/connector/charsets.py -> build/lib.linux-x86_64-2.7/mysql/connector
    copying lib/mysql/connector/cursor.py -> build/lib.linux-x86_64-2.7/mysql/connector
    copying lib/mysql/connector/version.py -> build/lib.linux-x86_64-2.7/mysql/connector
    copying lib/mysql/connector/authentication.py -> build/lib.linux-x86_64-2.7/mysql/connector
    copying lib/mysql/connector/custom_types.py -> build/lib.linux-x86_64-2.7/mysql/connector
    copying lib/mysql/connector/protocol.py -> build/lib.linux-x86_64-2.7/mysql/connector
    copying lib/mysql/connector/optionfiles.py -> build/lib.linux-x86_64-2.7/mysql/connector
    copying lib/mysql/connector/pooling.py -> build/lib.linux-x86_64-2.7/mysql/connector
    creating build/lib.linux-x86_64-2.7/mysql/connector/locales
    copying lib/mysql/connector/locales/__init__.py -> build/lib.linux-x86_64-2.7/mysql/connector/locales
    creating build/lib.linux-x86_64-2.7/mysql/connector/locales/eng
    copying lib/mysql/connector/locales/eng/__init__.py -> build/lib.linux-x86_64-2.7/mysql/connector/locales/eng
    copying lib/mysql/connector/locales/eng/client_error.py -> build/lib.linux-x86_64-2.7/mysql/connector/locales/eng
    creating build/lib.linux-x86_64-2.7/mysql/connector/django
    copying lib/mysql/connector/django/schema.py -> build/lib.linux-x86_64-2.7/mysql/connector/django
    copying lib/mysql/connector/django/client.py -> build/lib.linux-x86_64-2.7/mysql/connector/django
    copying lib/mysql/connector/django/__init__.py -> build/lib.linux-x86_64-2.7/mysql/connector/django
    copying lib/mysql/connector/django/introspection.py -> build/lib.linux-x86_64-2.7/mysql/connector/django
    copying lib/mysql/connector/django/validation.py -> build/lib.linux-x86_64-2.7/mysql/connector/django
    copying lib/mysql/connector/django/creation.py -> build/lib.linux-x86_64-2.7/mysql/connector/django
    copying lib/mysql/connector/django/features.py -> build/lib.linux-x86_64-2.7/mysql/connector/django
    copying lib/mysql/connector/django/operations.py -> build/lib.linux-x86_64-2.7/mysql/connector/django
    copying lib/mysql/connector/django/base.py -> build/lib.linux-x86_64-2.7/mysql/connector/django
    copying lib/mysql/connector/django/compiler.py -> build/lib.linux-x86_64-2.7/mysql/connector/django
    creating build/lib.linux-x86_64-2.7/mysql/connector/fabric
    copying lib/mysql/connector/fabric/__init__.py -> build/lib.linux-x86_64-2.7/mysql/connector/fabric
    copying lib/mysql/connector/fabric/balancing.py -> build/lib.linux-x86_64-2.7/mysql/connector/fabric
    copying lib/mysql/connector/fabric/connection.py -> build/lib.linux-x86_64-2.7/mysql/connector/fabric
    copying lib/mysql/connector/fabric/caching.py -> build/lib.linux-x86_64-2.7/mysql/connector/fabric
    creating build/lib.linux-x86_64-2.7/mysqlx
    copying lib/mysqlx/__init__.py -> build/lib.linux-x86_64-2.7/mysqlx
    copying lib/mysqlx/constants.py -> build/lib.linux-x86_64-2.7/mysqlx
    copying lib/mysqlx/errorcode.py -> build/lib.linux-x86_64-2.7/mysqlx
    copying lib/mysqlx/statement.py -> build/lib.linux-x86_64-2.7/mysqlx
    copying lib/mysqlx/connection.py -> build/lib.linux-x86_64-2.7/mysqlx
    copying lib/mysqlx/result.py -> build/lib.linux-x86_64-2.7/mysqlx
    copying lib/mysqlx/errors.py -> build/lib.linux-x86_64-2.7/mysqlx
    copying lib/mysqlx/crud.py -> build/lib.linux-x86_64-2.7/mysqlx
    copying lib/mysqlx/protobuf.py -> build/lib.linux-x86_64-2.7/mysqlx
    copying lib/mysqlx/charsets.py -> build/lib.linux-x86_64-2.7/mysqlx
    copying lib/mysqlx/compat.py -> build/lib.linux-x86_64-2.7/mysqlx
    copying lib/mysqlx/authentication.py -> build/lib.linux-x86_64-2.7/mysqlx
    copying lib/mysqlx/dbdoc.py -> build/lib.linux-x86_64-2.7/mysqlx
    copying lib/mysqlx/expr.py -> build/lib.linux-x86_64-2.7/mysqlx
    copying lib/mysqlx/protocol.py -> build/lib.linux-x86_64-2.7/mysqlx
    creating build/lib.linux-x86_64-2.7/mysqlx/locales
    copying lib/mysqlx/locales/__init__.py -> build/lib.linux-x86_64-2.7/mysqlx/locales
    creating build/lib.linux-x86_64-2.7/mysqlx/locales/eng
    copying lib/mysqlx/locales/eng/__init__.py -> build/lib.linux-x86_64-2.7/mysqlx/locales/eng
    copying lib/mysqlx/locales/eng/client_error.py -> build/lib.linux-x86_64-2.7/mysqlx/locales/eng
    running build_ext
    # Python architecture: 64-bit
    # Python ARCH_64BIT: True
    # Protobuf include directory: /usr/include/google/protobuf
    # Protobuf library directory: /usr/lib/x86_64-linux-gnu
    # Protobuf protoc binary: /usr/bin/protoc
    # Running protoc command: /usr/bin/protoc -I /tmp/pip-build-3o9xdQ/mysql-connector/src/mysqlxpb/mysqlx/protocol /tmp/pip-build-3o9xdQ/mysql-connector/src/mysqlxpb/mysqlx/protocol/mysqlx_datatypes.proto /tmp/pip-build-3o9xdQ/mysql-connector/src/mysqlxpb/mysqlx/protocol/mysqlx_expr.proto /tmp/pip-build-3o9xdQ/mysql-connector/src/mysqlxpb/mysqlx/protocol/mysqlx_notice.proto /tmp/pip-build-3o9xdQ/mysql-connector/src/mysqlxpb/mysqlx/protocol/mysqlx_sql.proto /tmp/pip-build-3o9xdQ/mysql-connector/src/mysqlxpb/mysqlx/protocol/mysqlx_expect.proto /tmp/pip-build-3o9xdQ/mysql-connector/src/mysqlxpb/mysqlx/protocol/mysqlx_resultset.proto /tmp/pip-build-3o9xdQ/mysql-connector/src/mysqlxpb/mysqlx/protocol/mysqlx_connection.proto /tmp/pip-build-3o9xdQ/mysql-connector/src/mysqlxpb/mysqlx/protocol/mysqlx.proto /tmp/pip-build-3o9xdQ/mysql-connector/src/mysqlxpb/mysqlx/protocol/mysqlx_session.proto /tmp/pip-build-3o9xdQ/mysql-connector/src/mysqlxpb/mysqlx/protocol/mysqlx_crud.proto --cpp_out=/tmp/pip-build-3o9xdQ/mysql-connector/src/mysqlxpb/mysqlx
    error: [Errno 2] No such file or directory
```
Also, perhaps this might not be available under Alpine, also (not sure).